src/pages/Lend/Filter/FilterComponents/FilterRangeSlider.vue

```diff
@@ -8,141 +8,92 @@
 				({{ rangeLabel }})
 			</span>
 		</div>
-		<div class="slider">
+		<div
+			class="slider"
+			v-if="range.min !== range.max"
+		>
 			<input
 				type="range"
-				:min="minimum"
-				:max="maximum"
+				:min="sliderMinimum"
+				:max="sliderMaximum"
 				:value="sliderValue"
 				:style="sliderStyle"
-				@input="onSliderChange(
-					refine,
-					range,
-					$event)">
-
-			<div class="hide">
-				<!-- TODO: create new component to replace this hidden element computed property -->
-				{{ sliderPosition }}
-			</div>
+				@input="onSliderChange(refine, range, $event)"
+			>
 		</div>
+		<div class="slider-unavailable" v-else>No refinements available</div>
```

Comment:
I added this because when the max and min of the slider are the same, the grabber always moves to the left and slider behavior becomes odd.